3542/3/views/static_pages/home.haml

```diff
@@ -0,0 +1,4 @@
+- if login?
```

Comment:
this seems to be routing logic - it should be in Controller not View.
View is responsible only for presenting object to user